### ISSUE TITLE
support $var() for json in key

### ISSUE THREAD

**Issue by seanchann** [OpenSIPS/opensips#1382]
hi, 
   I used below grammar in cfg file, it alway printf null in this key. if direct call `$json(test/deep1/deep2)` will work fine. whether there not support use $var in json key except the index of array:
   ```
   $var(test_key)="test/deep1/deep2"
   xlog("L_ERR", "$json($var(test_key))");
   ```

**Comment by bogdan-iancu** [comment]
yeah, the $json supports only static values as name, no variable accepted there. Let's have this as a feature request

**Comment by vtzan** [comment]
+1 for this feature. 
Thank you guys

**Comment by liviuchircu** [comment]
@seanchann @vtzan , are your JSONs that unstructured that you cannot even tell their nesting levels? For example, using variables inside `$json` like below will work:

    $json(foo) := "{\"a\": {\"c\": 7}, \"b\": 4}";
    $var(bar) = "a";
    $var(baz) = "c";

    xlog("XXX $json(foo/$var(bar)/$var(baz))\n");

As Bogdan said, we still cannot throw `"foo"` into a variable.

**Comment by seanchann** [comment]
@liviuchircu yes,  we need `foo` into a variable In some scenarios
